### ISSUE TITLE
Add Instructions on cloning, updating, and managing my local repo to README.md

### ISSUE THREAD

**Issue by CooperStansbury** [betteridiot/b575f19#7]
**Problem:** It is not obvious how to keep my local copy of the repository up to date with the online class resource. I'd like to be able to access new material on the online class resource without having to delete my folder and re-clone. 

**Solution:** A new section in [`README.md`](./README.md) that describes how to (1) obtain a local copy of the repository, (2) how to update my local copy when the online class resource changes, (3) a few links to helpful resources if we get stuck (4) common errors and how to resolve them. 

**Alternatives:** A canvas document.

**Comment by betteridiot** [comment]
You are correct that this needs to be fleshed out more, but the intention was to cover this in class (by design). However, there is already a section of the [README.md](https://github.com/betteridiot/b575f19#git-pull-error) that addresses the error of local changes preventing a `git pull`.

**Comment by CooperStansbury** [comment]
Cool beans. Perhaps adding something like this to the end of the [README.md](README.MD):

------------


# A Gentle Introduction to GitHub
This section will cover basic operations from the command line, and assumes that the reader is familiar with 2. and 3. from the Table on Contents (above).

## Cloning the Repository
1. First, navigate to the directory that you would like to clone the repository into using commands like:
   - `cd`: to change your working directory.
   - `cd ../` to "go back" a level in your directory tree.
2. Next, clone the remote repository and create a local copy on your machine using this command:
  - `git clone https://github.com/betteridiot/b575f19`
3. Now you can see the contents of the remote repository on your machine by moving into the newly cloned directory:
  - `cd b575f19/`
  - `ls -la`

## Updating Your Local Repository
When changes are made to the remote repository, they will not automatically sync with your local copy. You can update your local directory by running the following command **when your working directory is `b575f19`**. You can check your working directory by running `pwd`, the last item in the path should be `b575f19/`.

To update your local git directory:
  -`git pull`

**Note on `git pull` errors:** You may get a `git pull` error if you have made changes to files in your local git directory. This is ok, and normal. If you get an error when you `pull` (or update) the repo that says something like "Your local changes to the following files will be overwritten by merge’", you can do a `git stash` to address this. To find out how to do this, go to https://git-scm.com/book/en/v1/Git-Tools-Stashing to read all about it.

**Comment by betteridiot** [comment]
This is great, but also somewhat covered on this [wiki](https://github.com/betteridiot/b575f19/wiki/Updating-your-local-repo) in the repo. I just haven't added it to the README.md

**Comment by betteridiot** [comment]
However, feel free to give a PR on this and I would be happy to merge it in!
